File: 2108_return_first_palindrom/return_first_palindrome.py
# Py3: https://leetcode.com/problems/find-first-palindromic-string-in-the-array/?envType=daily-question&envId=2024-02-13
from typing import List
def firstPalindrome(self, words: List[str]) -> str:
        for w in words:
            if w == w[::-1]:
                return w
        
        return ""

File: 2108_return_first_palindrom/return_first_palindrome.js
// JS: https://leetcode.com/problems/find-first-palindromic-string-in-the-array/?envType=daily-question&envId=2024-02-13
// TS: https://leetcode.com/problems/find-first-palindromic-string-in-the-array/?envType=daily-question&envId=2024-02-13
function firstPalindrome (words) {
    for(let i = 0; i < words.length; i++) {
        if (words[i] === words[i].split("").reverse().join("")) {
            return words[i];
        }
    }

    return "";
};